src/containers/Bird/Bird.interface.tsx

```diff
@@ -0,0 +1,8 @@
+import { IBird } from '@constans/interface/birds.interface'
+
+export interface IPropsBird {
+  bird: IBird
+  onBirdNameClick: (value: IBird) => void
+  designType: string
```

Comment:
```suggestion
  designType?: 'error' | 'right' | 'default'
```
Тип нужно указать корректный, а не обычная строка